src/WorkflowManager/appsettings.Local.json

```diff
@@ -0,0 +1,124 @@
+{
+  "WorkloadManagerDatabase": {
+    "ConnectionString": "mongodb://root:rootpassword@localhost:30017",
+    "DatabaseName": "WorkloadManager",
+    "WorkflowCollectionName": "Workflows",
+    "WorkflowInstanceCollectionName": "WorkflowInstances",
+    "PayloadCollectionName": "Payloads"
+  },
+  "WorkflowManager": {
+    "endpointSettings": {
+      "defaultPageSize": 10,
+      "maxPageSize": 1000000
+    },
+    "TaskManager": {
+      "concurrency": 1,
+      "plug-ins": {
+        "argo": "Monai.Deploy.WorkflowManager.TaskManager.Argo.ArgoPlugin, Monai.Deploy.WorkflowManager.TaskManager.Argo",
+        "aide_clinical_review": "Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview.AideClinicalReviewPlugin, Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview",
+        "test": "Monai.Deploy.WorkflowManager.TaskManager.TestPlugin.TestPlugin, Monai.Deploy.WorkflowManager.TaskManager.TestPlugin"
+      },
+      "meta-data": {
+        "argo": "Monai.Deploy.WorkflowManager.TaskManager.Argo.Repositories.ArgoMetadataRepository, Monai.Deploy.WorkflowManager.TaskManager.Argo",
+        "test": "Monai.Deploy.WorkflowManager.TaskManager.TestPlugin.Repositories.TestPluginRepository, Monai.Deploy.WorkflowManager.TaskManager.TestPlugin"
+      }
+    },
+    "storage": {
+      "serviceAssemblyName": "Monai.Deploy.Storage.MinIO.MinIoStorageService, Monai.Deploy.Storage.MinIO",
+      "settings": {
+        "endpoint": "minio:30090", // note edit C:\Windows\System32\drivers\etc\Hosts fand add  `127.0.0.1	minio`
```

Comment:
I dont think putting comments like this in json like this is valid.

Reply:
https://reqbin.com/json/5wzepqmt/json-comment-example#:~:text=Can%20I%20add%20a%20comment,elements%20will%20still%20be%20data.